src/lightning/pytorch/utilities/imports.py

```diff
@@ -30,6 +30,7 @@
 _RICH_AVAILABLE = package_available("rich") and compare_version("rich", operator.ge, "10.2.2")
 _TORCHVISION_AVAILABLE = RequirementCache("torchvision")
 _LIGHTNING_COLOSSALAI_AVAILABLE = RequirementCache("lightning-colossalai")
+_LIGHTNING_BAGUA_AVAILABLE = RequirementCache("lightning-bagua")
```

Comment:
These 2 should be defined in the accelerator connector, which is the only file that uses it.

Reply:
@carmocca In fact, there are other files that use `_LIGHTNING_COLOSSALAI_AVAILABLE`, such as `src/lightning/pytorch/callbacks/gradient_accumulation_scheduler.py` and `tests/tests_pytorch/callbacks/test_gradient_accumulation_scheduler.py`. However, `_LIGHTNING_BAGUA_AVAILABLE` is currently only used in `src/lightning/pytorch/trainer/connectors/accelerator_connector.py`.

@Borda @carmocca  Any suggestions to fix it?

Reply:
It's okay. I'll look into this in a follow-up